src/lib/pickles/wrap.ml

```diff
@@ -595,10 +593,7 @@ let%test_module "gate finalization" =
             Kimchi_bindings.Protocol.Proof.Fp.example_with_range_check0
 
         let actual_feature_flags =
-          { Plonk_types.Features.none_bool with
-            range_check0 = true
-          ; lookup = true
-          }
+          { Plonk_types.Features.none_bool with range_check0 = true }
```

Comment:
The ECDSA-Keccak converter (meaning, takes the keccak output and expresses it in ECDSA-friendly format) uses lookups of 12-bits in order to check some lengths. That's why we had to include `Lookup` in `plonk_constraint_system.ml`. Nonetheless, I wonder if this modification affects this behavior in any way, or will require changes to the way that function works (but that's in develop right now, only).

Reply:
This is the helper I am referring to:

https://github.com/MinaProtocol/mina/blob/bf27b1e9fed79d8d84855fbde047bab3f2a77348/src/lib/crypto/kimchi_backend/gadgets/foreign_field.ml#L1432

Reply:
Lookup gate, not lookup argument. It's correct now